crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -87,6 +87,12 @@ impl GlobalTransform {
         GlobalTransform(Affine3A::from_scale(scale))
     }
 
+    #[doc(hidden)]
+    #[inline]
+    pub fn from_isometry(iso: Isometry3d) -> Self {
```

Comment:
The caveats should be clearly listed on the docs, even for internal use.

Reply:
This one doesn't really have any caveats — only `to_isometry` does.